### ISSUE TITLE
ngx-loading-bar break unsuscribe

### ISSUE THREAD

**Issue by jeremieca** [aitboudad/ngx-loading-bar#33]
When using ngx-loading-bar on ```'@angular/http';```.

The request cancel on Obervable unsuscribe does not work.

```
this.getProducts
   .subscribe((products) => {
        ....
   })
   .unsuscribe();      // Does not cancel request when using ngx-loading-bar  !!!!!
```

Have you an idea about this problem ?

**Comment by aitboudad** [comment]
fixed in `1.0.0-alpha.18`

**Comment by jeremieca** [comment]
Thanks. It works.

Envoyé de mon iPhone
